homeassistant/components/light/xiaomi_miio.py

```diff
@@ -36,13 +39,33 @@
 
 SUCCESS = ['ok']
 ATTR_MODEL = 'model'
+ATTR_SCENE = 'scene'
+
+SERVICE_SET_SCENE = 'xiaomi_miio_set_scene'
+
+XIAOMI_MIIO_SERVICE_SCHEMA = vol.Schema({
+    vol.Optional(ATTR_ENTITY_ID): cv.entity_ids,
+})
+
+SERVICE_SCHEMA_SCENE = XIAOMI_MIIO_SERVICE_SCHEMA.extend({
+    vol.Required(ATTR_SCENE):
+        vol.All(vol.Coerce(int), vol.Clamp(min=1, max=4))
+})
+
+SERVICE_TO_METHOD = {
+    SERVICE_SET_SCENE: {
+        'method': 'async_set_scene',
+        'schema': SERVICE_SCHEMA_SCENE}
+}
```

Comment:
Remove this, for only one service we don't need a map and can add it later if that would be needed